### PR TITLE
tribbles

### PR DESCRIPTION
**Pull request by howard-stearns**
Drops a bunch of "tribbles" (small spheres) in front of you.
They edit change color at 60hz and jump randomly at an average 1hz for 30 seconds, and then dissapear after 60 seconds.
Useful for testing the load of physical objects.

### CODE REVIEW

**Comment by howard-stearns** [comment]
For testing, you need the main script (tribbles.js) to reference the entityScripts/tribble.js.

You can do this by editing tribble.js to point to a (possibly file) url that has the entity script (such as http://howard-stearns.github.io/models/scripts/load/entityScriptTribble.js), 
or test with the pre-edited copy at http://howard-stearns.github.io/models/scripts/load/tribbles.js

File: examples/tests/performance/tribbles.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            dimensions: {x: SIZE, y: SIZE, z: SIZE},
+            color: {red: Math.random() * 255, green: Math.random() * 255, blue: Math.random() * 255},
+            velocity: VELOCITY,
+            angularVelocity: Vec3.multiply(Math.random(), ANGULAR_VELOCITY),
+            damping: DAMPING,
+            angularDamping: ANGULAR_DAMPING,
+            gravity: GRAVITY,
+            collisionsWillMove: true,
+            lifetime: LIFETIME,
+            script: "https://s3.amazonaws.com/hifi-public/scripts/entityScripts/tribble.js"
```
Comment (imgntn) [inline examples/tests/performance/tribbles.js]: you could do a relative reference if you want ```Script.resolvePath(../../entityScripts/tribble.js```

**Comment by imgntn** [comment]
works for me windows 8

File: examples/tests/performance/tribbles.js (hunk 1 above)
Comment (howard-stearns) [inline examples/tests/performance/tribbles.js]: I haven't been confident enough that the definition of Script.resolvePath is stable enough, and a quick search did not reveal any other scripts doing this (for entityScripts).  But if you feel it's stable enough, I could do it either now or next time around...

**Pull request merged**